.circleci/.pypirc.enc

```diff
@@ -0,0 +1 @@
+Salted__���x0Ȏ��ji
����D
```

Comment:
Is it really that small (31 bytes)? In artman, it's 128 bytes. Just wondering if everything is there.

Reply:
All that's in there is the password; I'm passing the user via argv.

Reply:
I see, thank you!